README.md

```diff
@@ -86,6 +86,10 @@ The 'General' tests are designed to test any commodity CNF running on OpenShift,
 'CNF-specific' tests are designed to test some unique aspects of the CNF under test are behaving correctly.  This could
 include specifications such as issuing a `GET` request to a web server, or passing traffic through an IPSEC tunnel.
 
+## Turn off openshift required Tests when CNF run on kubernetes only environment
+when CNF test for CNF running on k8s only envirnment. execute commandline below on cli before compile tool and run shell script.
+   export TNF_MINIKUBE_ONLY=true
```

Comment:
I'll second what Marek has said. You can also see the rest of the documentation to see how other examples are formatted.